docs/getting-started.md

```diff
@@ -143,15 +143,21 @@ Install KubeVirt, Flannel and Cluster Network Operator:
 kubectl apply -f https://raw.githubusercontent.com/coreos/flannel/master/Documentation/kube-flannel.yml
 
 # Check for latest version
-RELEASE=v0.16.0
-kubectl apply -f https://github.com/kubevirt/kubevirt/releases/download/${RELEASE}/kubevirt.yaml
+
+# On other OS you might need to define it like
+export ="v0.18.0"
+# On Linux you can obtain it using 'curl' via:
+export KUBEVIRT_VERSION=$(curl -s https://api.github.com/repos/kubevirt/kubevirt/releases | grep tag_name | grep -v -- - | sort -V | tail -1 | awk -F':' '{print $2}' | sed 's/,//' | xargs)
+
+kubectl create -f https://github.com/kubevirt/kubevirt/releases/download/${KUBEVIRT_VERSION}/kubevirt-operator.yaml
```

Comment:
Add also the creation of the kubevirt-cr in order to deploy kubevirt resources by the operator.
```
kubectl create -f https://github.com/kubevirt/kubevirt/releases/download/${KUBEVIRT_VERSION}/kubevirt-cr.yaml
```

Reply:
it would probably a good decision to wait till kubevirt pods are available before moving to next step.

Reply:
@masayag done